core/src/components/title/test/basic/index.html

```diff
@@ -33,7 +33,7 @@
         </ion-header>
 
         <ion-header>
-          <ion-toolbar>
+          <ion-toolbar class="toolbar-title-default">
```

Comment:
We should look into why the incorrect class is being added here when the small title is hidden, but this shouldn't hold up this work. 👍